.github/workflows/release.yml

```
@@ -0,0 +1,51 @@
+name: Bump release version
+on:
+  pull_request:
+    branches: [rolling]
+    types:
+      - closed
+
+permissions:
+  contents: write
+
+concurrency: production
+
+jobs:
+  build:
+    name: bump tag version and release  
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.sha }}
+          fetch-depth: 0
+      - name: Bump version and push tag
+        id: version-bump
+        uses: hennejg/github-tag-action@v4.3.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          release_branches: rolling
+      - name: get-app-version
+        id: package-version
+        run: |
+          LF_VERSION=$(cat package.json | jq -r '.version')
+          echo "current-version=$LF_VERSION" >> "$GITHUB_OUTPUT"
```

```suggestion
        run: |
          LF_VERSION=$(cat Cargo.toml | grep -oP '^\s*version\s*=\s*"\K[^"]+' | head -n 1)
          echo "current-version=$LF_VERSION" >> "$GITHUB_OUTPUT"
```

Extracts version from `Cargo.toml` instead of `package.json`